incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -315,6 +316,69 @@ Similarly, if you want to make `ns-bar` a root again, you must be an
 administrator of the root namespace that is an ancestor of `ns-bar`, since the
 admins of that namespace will lose access to `ns-bar` once it becomes a root.
 
+
+<a name="move-subnamespace"/>
+
+### Move a Subnamespace
+
+Changing the parent of a subnamespace is not yet permitted in HNC, though there may be times
+where you want to move a subnamespace from one parent to another. 
+
+Consider the following tree:
+
+```
+org
+ - ns-team-1 (full namespace)
+   - ns-product-1 (subnamespace)
+   - ns-product-2 (subnamespace)
+ - ns-team-2 (full namespace)
+   - ns-product-3 (subnamespace)
+```
+
+We now want to move `ns-product-2` from `ns-team-1` to `ns-team-2`.
```

Comment:
Can we change these instructions as follows? To turn subns->full ns:

1. Prereqs: you must have "edit" permission on the subns and "delete" permission on anchors in the parent namespace.
2. Remove the `subnamespaceOf` annotation from the child.
3. Confirm that the anchor in the parent is now in the `Conflict` state.
4. Remove the anchor from the parent.

The namespace is now a full namespace. It remains a child of the same parent unless otherwise changed.

It's far more rare that you'll want to turn a full namespace into a subns - the only advantage of this is that users without cluster-level privileges will be able to _delete_ the namespace. But if you want to do this:

1. Ensure the full namespace has the correct parent.
2. Add the `subnamespaceOf` to the child, pointing to the current parent.
3. Create the anchor in the parent (`kubectl hns create child -n parent`). Confirm that the anchor has the `Ok` state.

The child is now a subnamespace, and will be deleted if the anchor is deleted.

You may want to double-check all these steps (and the exact format of the status messages) to ensure they're correct, I'm writing by memory.